pkg/generate/rand.go

```diff
@@ -70,3 +70,13 @@ func randomBytes(src rand.Source, bytes string, n int) []byte {
 
 	return b
 }
+
+// RandomInt returns a generated int in the range >= min, <= max
+func RandomInt() int {
+	var (
+		min = 1
+		max = 100
+	)
+	rand.Seed(time.Now().UnixNano())
+	return min + rand.Intn(max-min+1)
```

Comment:
Having `min` and `max` as variables would only help if they could change, in our case they constant, to the point where it might make sense to just inline the correct numbers as in the comment above.